torch/csrc/autograd/python_function.cpp

```diff
@@ -392,7 +392,8 @@ static void _wrap_outputs(
     for (const auto i : c10::irange(num_inputs)) {
       PyObject* input = nullptr;
       if (self->is_variable_input[i]) {
-        if (grad_inputs[variable_idx].defined() || !self->materialize_grads) {
+        if (grad_inputs[variable_idx].defined() || !self->materialize_grads ||
```

Comment:
Need to update the docs of ctx.set_materialize_grads. It doesn't say anything about the JVP case: https://pytorch.org/docs/stable/generated/torch.autograd.function.FunctionCtx.set_materialize_grads.html

Reply:
Updated.